src/DataType/OccurrenceDateTime.php

```diff
@@ -31,4 +31,16 @@ public function getFormat(): string
     {
         return $this->format;
     }
+
+    public function toXML(\DOMDocument $document): \DOMElement
+    {
+        $element = $document->createElement('ram:OccurrenceDateTime');
+
+        $dateTimeElement = $document->createElement('ram:DateTimeString', $this->dateTimeString->format('Ymd'));
+        $dateTimeElement->setAttribute('format', '102');
```

Comment:
For this attribute, I would prefer to use the "format" attribute to avoid potential issues later.